sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -1,8 +1,25 @@
 # Release History
 
+## 1.2.0
+
+### Features Added
+- Added support to customize the Communication Identity access token�s validity period:
+    - Added method overloads that provide the ability to create a Communication Identity access token with custom expiration:
+        - `CreateUserAndToken(IEnumerable<CommunicationTokenScope> scopes, TimeSpan tokenExpiresAfter, CancellationToken cancellationToken = default)`
+        - `CreateUserAndTokenAsync(IEnumerable<CommunicationTokenScope> scopes, TimeSpan tokenExpiresAfter, CancellationToken cancellationToken = default)`
+        - `GetToken(CommunicationUserIdentifier communicationUser, IEnumerable<CommunicationTokenScope> scopes, TimeSpan tokenExpiresAfter, CancellationToken cancellationToken = default)`
+        - `GetTokenAsync(CommunicationUserIdentifier communicationUser, IEnumerable<CommunicationTokenScope> scopes, TimeSpan tokenExpiresAfter, CancellationToken cancellationToken = default)`
+- Added a new API version `ServiceVersion.V2022_10_01` that is now the default API version.
+
 ## 1.2.0-beta.1 (Unreleased)
 
+## 1.1.0 (Unreleased)
```

Comment:
can you please merge the latest `main` into this PR to get a fresh view?

Reply:
Updated.